pages/index.tsx

```diff
@@ -0,0 +1,562 @@
+import { FunctionComponent } from 'react';
+
+import { H1, H3 } from '../styles/MDComponents';
+import PageLink, { CallToAction } from '../components/PageLink';
+
+import AnalyticsManagerIcon from '../images/svg/analyticsmanager.svg';
+import DevIcon from '../images/svg/dev.svg';
+
+import styles from './index.module.scss';
+
+function emptyRowElement() {
+  return <div style={{ display: 'flex', padding: '15px' }} />;
+}
+
+const LandingPage: FunctionComponent = () => {
+  return (
+    <div style={{ width: '85%' }}>
+      <H1>Getting started with Avo</H1>
+      <p className={styles.text}>
+        We distinguish 3 levels of Avo adoption, you don&apos;t need to go all
+        the way, because every stage has value in itself.
+      </p>
+
+      <div className={styles.tocBg}>
+        <p>
+          <a
+            href="#stage1"
+            className={[styles.tocItem, styles.avoPink].join(' ')}
+          >
+            1. Better tracking plan management today - no code solution
+          </a>
+        </p>
+        <p>
+          <a
+            href="#stage2"
+            className={[styles.tocItem, styles.avoPink].join(' ')}
+          >
+            2. Better tracking validation
+          </a>
+        </p>
+        <p>
+          <a
+            href="#stage3"
+            className={[styles.tocItem, styles.avoPink].join(' ')}
+          >
+            3. Better tracking implementation
+          </a>
+        </p>
+        <p>
+          <a
+            href="#workspace"
+            className={[styles.tocItem, styles.avoGrey].join(' ')}
+          >
+            Explore the Avo workspace
+          </a>
+        </p>
+        <p>
+          <a
+            href="#guides"
+            className={[styles.tocItem, styles.avoGrey].join(' ')}
+          >
+            Comprehensive guides
+          </a>
+        </p>
+      </div>
+
+      <p className={styles.text}>
+        For example, if you already have a framework for analytics
+        implementation and implementation data bugs are not a problem for you,
+        you can stay at the second adoption stage.
+      </p>
+      <h2 id="stage1">
+        1. Better tracking plan management today - no code solution
+      </h2>
+      <p className={styles.text}>
+        Use Avo instead of your spreadsheet to manage and discuss your tracking
+        plan. Avo is designed for analytics, so it&apos;s way better than the
+        spreadsheet for that use case.
```

Comment:
```suggestion
        Use Avo instead of your spreadsheet to manage and discuss your tracking
        plan. Avo is designed for analytics, so it&apos;s way better than the
        spreadsheet for managing your tracking plan.
```